packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -367,6 +369,23 @@ export const AddressStruct: Describe<AddressElement> = element('Address', {
   avatar: optional(boolean()),
 });
 
+/**
+ * A struct for the {@link AccountSelectorElement} type.
+ */
+export const AccountSelectorStruct: Describe<AccountSelectorElement> = element(
+  'AccountSelector',
+  {
+    name: string(),
+    hideExternalAccounts: optional(boolean()),
+    chainIds: optional(array(CaipChainIdStruct)) as unknown as Struct<
```

Comment:
Why does this need a type cast?

Reply:
![image](https://github.com/user-attachments/assets/2600c10d-3770-42f0-a09f-f47c5e3cc300)